### PR TITLE
Added support for multiple endpoints in WebsiteMonitor

### PR DESCRIPTION
**Pull request by jotes**
@pwilczynskiclearcode could you look at this?

### CODE REVIEW

File: src/collectors/websitemonitor/websitemonitor.py [hunk 1]
```diff
@@ -72,7 +73,11 @@ def collect(self):
...
         except IOError, e:
-            self.log.error('Unable to open %s' % (self.config['URL']))
+            self.log.error('Unable to open %s' % (endpoint))
 
         except Exception, e:
             self.log.error("Unknown error opening url: %s", e)
+
+    def collect(self):
+        for endpoint in self.config['endpoints']:
+            self.check_endpoint(endpoint)
```
Comment (pwilczynskiclearcode) [inline src/collectors/websitemonitor/websitemonitor.py:92]: does it produce separate metrics?

**Comment by pwilczynskiclearcode** [comment]
:+1:

**Comment by jotes** [comment]
Tests are passing :D good point, i'll look into it.

On Fri, Nov 13, 2015 at 3:29 PM, Paweł Wilczyński notifications@github.com
wrote:

> [image: :+1:]
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/ClearcodeHQ/Diamond/pull/1#issuecomment-156447296.

## 

jarek@reijutsu:~$ fortune
You have Egyptian flu: you're going to be a mummy.
jarek@reijutsu:~$ fortune
You now have Asian Flu.

File: src/collectors/websitemonitor/websitemonitor.py (hunk 1 above)
Comment (jotes) [inline src/collectors/websitemonitor/websitemonitor.py:92]: yup, now it does.

**Pull request merged**